pkg/console/starter/starter.go

```diff
@@ -108,13 +112,36 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
-	recorder := ctx.EventRecorder
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		api.OpenshiftConfigManagedNamespace,
+		api.OpenShiftConsoleOperatorNamespace,
+	)
 
 	operatorClient := &operatorclient.OperatorClient{
 		Informers: operatorConfigInformers,
 		Client:    operatorConfigClient.OperatorV1(),
 	}
 
+	resourceSyncer := resourcesynccontroller.NewResourceSyncController(
```

Comment:
I don't think we need the resource sync at all. We're not mounting the config map into a pod. We should be able to read it in place.

Reply:
Refactored code to not use the resourcesync.  Added new informer for the ConfigMaps in the `openshift-config-managed` namesapce and trigger off that to update the console config.